docs/guide/time.rst

```diff
@@ -16,29 +16,37 @@ SunPy builds upon its functionality.
 
 Solar data is associated with a number of different time formats. SunPy provides a simple
```

Comment:
The opening paragraph to this file talks about `datetime.datetime` it should be updated.